app/docs/dev/policies/meshaccesslog.md

```diff
@@ -0,0 +1,283 @@
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficLog](/docs/{{ page.version }}/policies/traffic-log).
+{% endwarning %}
+
+{% tip %}
+In the rest of this page we assume you have already configured your observability tools to work with Kuma.
+If you haven't already read the [observability docs](/docs/{{ page.version }}/explore/observability).
+{% endtip %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+|-------------------|-----------|-----|------|
+| Mesh              | ✅         | ✅   | ✅    |
+| MeshSubset        | ✅         | ❌   | ❌    |
+| MeshService       | ✅         | ✅   | ❌    |
+| MeshServiceSubset | ✅         | ❌   | ❌    |
+
+If you don't understand this table you should read [matching docs](/docs/{{ page.version }}/policies/matching).
+
+## Configuration
+
+### Format
+
+Kuma gives you full control over the format of the access logs.
+
+The shape of a single log record is defined by a template string that uses [command operators](https://www.envoyproxy.io/docs/envoy/v1.22.0/configuration/observability/access_log/usage#command-operators) to extract and format data about a `TCP` connection or an `HTTP` request.
+
+E.g.,
+
+```
+%START_TIME% %KUMA_SOURCE_SERVICE% => %KUMA_DESTINATION_SERVICE% %DURATION%
+```
+
+where `%START_TIME%` and `%KUMA_SOURCE_SERVICE%` are examples of available _command operators_.
+
+All _command operators_ [defined by Envoy](https://www.envoyproxy.io/docs/envoy/v1.22.0/configuration/observability/access_log/usage#command-operators) are supported, along with additional _command operators_ defined by Kuma:
+
+| Command Operator                     | Description                                                      |
+|--------------------------------------|------------------------------------------------------------------|
+| `%KUMA_MESH%`                        | name of the mesh in which traffic is flowing                     |
+| `%KUMA_SOURCE_SERVICE%`              | name of a `service` that is the `source` of traffic              |
+| `%KUMA_DESTINATION_SERVICE%`         | name of a `service` that is the `destination` of traffic         |
+| `%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%` | address of a `Dataplane` that is the `source` of traffic         |
+| `%KUMA_TRAFFIC_DIRECTION%`           | direction of the traffic, `INBOUND`, `OUTBOUND` or `UNSPECIFIED` |
+
+All additional access log _command operators_ are valid to use with both `TCP` and `HTTP` traffic.
+
+If a _command operator_ is specific to `HTTP` traffic, such as `%REQ(X?Y):Z%` or `%RESP(X?Y):Z%`, it will be replaced by a symbol "`-`" in case of `TCP` traffic.
+
+Internally, Kuma [determines traffic protocol](/docs/{{ page.version }}/policies/protocol-support-in-kuma) based on the value of `kuma.io/protocol` tag on the `inbound` interface of a `destination` `Dataplane`.
```

Comment:
In the old policy we had a section here:

```
% tip %

To provide different format for TCP and HTTP logging you can define two `MeshAccessLog` entities, one for TCP and one for HTTP with matching `kuma.io/protocol` selector.

% endtip %
```

But if I understand the matching matrix right this won't work yet because we don't support MeshServiceSubset. @michaelbeaumont - can you confirm?

Reply:
Yeah right now I think `MeshServiceSubset` is rejected via validation so we can leave this out

Reply:
Right now I think `MeshServiceSubset` is rejected via validation so we can leave this out

Reply:
Yeah that's where we need some tags to be service specific not inbound specific :(